helpers/helpers.go

```diff
@@ -153,6 +156,75 @@ func GetIncludedBundles(filename string) ([]string, error) {
 	return includes, nil
 }
 
+// UnpackFile unpacks a .tar or .tar.gz/.tgz file to a given directory.
+// Should be roughly equivalent to "tar -x[z]f file -C dest". Does not
+// overwrite; returns error if file being unpacked already exists.
+func UnpackFile(file string, dest string) error {
+	fr, err := os.Open(file)
+	if err != nil {
+		return err
+	}
+	defer func() {
+
```

Comment:
Minor: you can drop this extra newline.

Reply:
Weird. Thanks for catching that. Fixed in forthcoming push.